src/main/java/picard/sam/SetNmMdAndUqTags.java

```diff
@@ -73,6 +73,9 @@
     @Option(doc = "Whether the file contains bisulfite sequence (used when calculating the NM tag).")
     public boolean IS_BISULFITE_SEQUENCE = false;
 
+    @Option(doc = "Only set the UQ tag, ignore MD and NM")
+    public boolean SET_ONLY_UQ = false;
```

Comment:
Is there a use case for individually doing nm or md or those two without uq?  Or if we plan on making this applicable to more tags then it should be a different clp altogether.  With this list I would then have to do something like 
```
if (list.contains("nm"){
     stream.peek....
} else if (list.contains("md") {
etc. etc.
```
no?  If you think there is a use case that warrants this then I can change it or if there is a better way to write the above.